src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -42,6 +42,34 @@ define(function (require, exports, module) {
     var STEP_MULTIPLIER = 5;
 
     /**
+     * Convert 0x notation into hex6 format for tinycolor
+     * compatibility: ("0xFFAACC" => "#FFFFFF")
+     */
+    function color0xToHex(color,toStr){
+        var format0xToHexColor = color.replace("0x","#");
+        var hexColor = tinycolor(format0xToHexColor);
+        hexColor._format = "0x";
+
+        if(toStr !== null){
+            hexColor.toString();
```

Comment:
what does this do?
I'd expect `toString();` result to be assigned to something?
or did you mean to do `return hexColor.toString();` ?